worlds/messenger/test/test_shop_chest.py

```diff
@@ -50,14 +50,14 @@ def testSealsAmount(self) -> None:
         self.assertEqual(len(required_seals), 22)
 
 
-class ThirtyThirtySeals(MessengerTestBase):
+class ThirtyThirtySeals(HalfSealsRequired):
```

Comment:
I think I did it because I convinced myself I could get an actual benefit from it, but it ended up not being the case, and I just forgot to change it back